keg/utils.py

```diff
@@ -112,7 +112,9 @@ def pymodule_fpaths_to_objects(fpaths):
             with open(fpath) as fo:
                 exec(compile(fo.read(), fpath, 'exec'), pymodule_globals)
             retval.append((fpath, pymodule_globals))
-        except (FileNotFoundError, PermissionError):
+        except FileNotFoundError:
```

Comment:
Why are you using two separate `except` blocks instead of one with all exceptions?  

Let's return a named tuple from this with three attributes `fpath, data, error` set appropriately.  Then update calling code to "do the right thing" in case of success _and_ error.

Reply:
Discussing with @guruofgentoo, we decided to ignore FileNotFoundErrors entirely, since this function is only searching _potential_ config paths. The searched paths can be viewed via `develop config` in the keg CLI. We also decided to ignore permission errors during app startup, but show unreadable paths when running `develop config`.